models/errors.js

```diff
@@ -28,6 +28,13 @@ module.exports = {
             this.status = 404;
         }
     },
+    MissingAttributeError: class MissingAttributeError extends Error {
```

Comment:
oooh great add, guessing this will be used a lot during error handling in the code